easybuild/easyblocks/s/suitesparse.py

```diff
@@ -167,8 +167,8 @@ def make_module_req_guess(self):
         """
         guesses = super(EB_SuiteSparse, self).make_module_req_guess()
         guesses.update({
-            'CPATH': [self.config_name],
-            'LD_LIBRARY_PATH': ['UMFPACK/Lib', 'AMD/Lib'],
+            'CPATH': ['include', self.config_name],
+            'LD_LIBRARY_PATH': ['lib', 'UMFPACK/Lib', 'AMD/Lib'],
```

Comment:
how about changing it to:

```python
guesses['CPATH'].append(self.config_name)

extra_libs = ['AMD/Lib', 'UMFPACK/Lib']
guesses['LD_LIBRARY_PATH'].extend(extra_libs)
guesses['LIBRARY_PATH'].extend(extra_libs)
```

extra bonus points :cookie: if you also tackle the issue reported in #882 while you're at it :-)